godot_project/editor/ring_menu_levels/help_menu/ring_level_help_menu.gd

```diff
@@ -0,0 +1,17 @@
+class_name RingLevelHelpMenu extends RingMenuLevel
+
+
+var _workspace_context: WorkspaceContext = null
+var _ring_menu: NanoRingMenu = null
+
+
+func _init(in_workspace_context: WorkspaceContext, in_menu: NanoRingMenu) -> void:
+	_workspace_context = in_workspace_context
+	_ring_menu = in_menu
+	super._init(
+		[],
+		tr("Help"),
+		tr("Find Help")
+	)
+	add_action(preload("ring_action_about_msep.gd").new(in_workspace_context, in_menu))
```

Comment:
those actions are usually stored in _workspace_context, is there anything different with this one to be treated differently? If there is no reason behind breaking this notation then I would suggest to keep it the same way for consistency

Reply:
we put in _workspace_context only ring actions that we want to access from somewhere else, since this action is so simple the code in execute() can be duplicated instead of executing the ring action